admin/src/app/account.service.ts

```diff
@@ -1,25 +1,59 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs';
+import { tap } from 'rxjs/operators';
 
 import { environment } from 'src/environments/environment';
+import { AuthService } from './auth.service';
+import { ACCESS_TOKEN_KEY } from './Core/getToken';
 
 @Injectable({
   providedIn: 'root'
 })
 export class AccountService {
   private readonly apiUrl: string;
-  
+
   public constructor(
-    private readonly http: HttpClient
+    private readonly http: HttpClient,
+    private readonly authService: AuthService
   ) { 
     this.apiUrl = environment.api;
   }
+    
+  public isAdmin(): boolean {
+    return this.authService.role() === 'admin';
+  }
+
+  public isClient(): boolean {
+    return this.authService.role() === 'client';
+  }
+
+  public id(): number | null {
```

Comment:
Getting this value property looks like accountService.id(). It seems clear for me that this is account id. We may discuss this point

Reply:
We have discussed this point with Andrew. If I have account service, then I have some account resource and id is this account resource id. The main point is about what is account resource in my application

I still think that it would be better to divide services if I have something like organization and put organization id inside organization service. 

 If you insist I change id naming, but right now it seems to be more appropriate to use accountService.id